library/std/src/os/unix/net/listener.rs

```diff
@@ -73,10 +73,10 @@ impl UnixListener {
         unsafe {
             let inner = Socket::new_raw(libc::AF_UNIX, libc::SOCK_STREAM)?;
             let (addr, len) = sockaddr_un(path.as_ref())?;
-            #[cfg(target_os = "linux")]
+            #[cfg(any(target_os = "linux", target_os = "freebsd"))]
             const backlog: libc::c_int = -1;
```

Comment:
```suggestion
            const backlog: libc::c_int = if cfg!(any(target_os = "linux", target_os = "freebsd")) { -1 } else { 128 };
```

(Not sure if rustfmt will want to wrap this or not.)